traffic_ops/app/db/seeds.sql

```diff
@@ -236,6 +214,7 @@ CROSS JOIN ( VALUES
 	('DIVISION:UPDATE'),
 	('DNS-SEC:READ'),
 	('DNS-SEC:UPDATE'),
+    ('DNS-SEC:DELETE')
```

Comment:
Missing comma, which is why the tests failed.

Also you're using spaces instead of indentation